util/random.go

```diff
@@ -8,12 +8,9 @@ import (
 	"github.com/segmentio/ksuid"
 )
 
-func init() {
-	rand.NewSource(time.Now().UnixNano())
-}
-
 // RandomString generates a random string of length n.
 func RandomString(n int) string {
+	rand.NewSource(time.Now().UnixNano())
```

Comment:
Why was this moved out of init?

Reply:
Since it was in the init, when the function was reused it kept getting the same result.